src/modules/backlight.cpp

```diff
@@ -76,6 +80,25 @@ namespace modules {
     return true;
   }
 
+  string backlight_module::get_output() {
+    // Get the module output early so that
+    // the format prefix/suffix also gets wrapped
+    // with the cmd handlers
+    string output{module::get_output()};
+
+    if (m_scroll) {
+      m_builder->cmd(mousebtn::SCROLL_UP, EVENT_SCROLLUP);
+      m_builder->cmd(mousebtn::SCROLL_DOWN, EVENT_SCROLLDOWN);
+    }
+
+    m_builder->append(output);
```

Comment:
```suggestion
    m_builder->append(std::move(output));
```

Just add a merge here and it'll be ready to be merged for me.